main/src/ca/uwaterloo/flix/language/phase/jvm/BackendType.scala

```diff
@@ -187,6 +187,8 @@ object BackendType {
       case MonoType.RecordEmpty => BackendObjType.RecordEmpty.toTpe
       case MonoType.RecordExtend(_, value, _) => BackendObjType.RecordExtend(toBackendType(value)).toTpe
       case MonoType.Native(clazz) => BackendObjType.Native(JvmName.ofClass(clazz)).toTpe
+      case MonoType.JvmConstructor(method) => BackendObjType.JavaObject.toTpe
```

Comment:
MonoType should not have JvmConstructor or JvmMethod :)

So these changes can be undone.

Reply:
Thank you.